### ISSUE TITLE
Crash on Login if tap Sign in button several times

### ISSUE THREAD

**Issue by annadanchenko** [status-im/status-react#871]
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a user, I want to login so that I can use the app.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: When user tries to log in and taps Sign in button several times then app crashes

#### Expected behavior
Typed in password, tap on Sign in button, tap on Sign in button (as there is no feedback from the app and user might think that he missed the button first time) -> home screen is shown

#### Actual behavior
Typed in password, taped 2-3 times on Sign in button, after this Home screen is shown for a sec and then app crashed

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Tap on username to log in
- In Password field type valid password for this user
- Tap on button Sign in 2-3 times

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- [ ] fix it

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: 0.9.3

* Operating System: iOS 10.2.1
* Device: iPhone 7 plus
* Instabug: 
  * https://dashboard.instabug.com/applications/status/beta/bugs/131
  * https://dashboard.instabug.com/applications/status/beta/bugs/133
  * https://dashboard.instabug.com/applications/status/beta/bugs/159

#### Logs

[comment]: # (if on Android please replicate bug whilst running adb logcat)
[console1.txt](https://github.com/status-im/status-react/files/828191/console1.txt)
[StatusIm-2017-03-08-170105.ips.beta.txt](https://github.com/status-im/status-react/files/828192/StatusIm-2017-03-08-170105.ips.beta.txt)

**Comment by commiteth** [comment]
![Contract deploying](https://commiteth.com/img/deploying_contract.png)

**Comment by coyotespike** [comment]
I was able to reproduce once, immediately after starting Simulator and on first login.
```
console.error: "Attempted to transition from state 'RESPONDER_ INACTIVE_ PRESS_ IN' 
to 'RESPONDER_ ACTIVE_ LONG_ PRESS_ IN', which is not supported. 
This is most likely due to 'Touchable.longPressDelayTimeout' not being cancelled."
```

After I logged in and then tried to switch users, I couldn't reproduce, even if I quit iOS Simulator and ran `react-native run-ios` again. I'll update if I figure out how to reproduce reliably.

Here is the best discussion of this issue: https://github.com/facebook/react-native/issues/1693
TL;DR - it's probably an RN issue related to component timeouts.

**Comment by annadanchenko** [comment]
Issue can be reproduced on 0.9.6 with same steps: Tap 2-3 times on Sign In button. App does not immediately crash. It takes several seconds, so user can open chats etc and then crash happens. 
video: https://drive.google.com/open?id=0Bz3t9zSg1wb7OHIza3Q2YnN4dkU
logs:
[20172404113738-consolelog.txt](https://github.com/status-im/status-react/files/951282/20172404113738-consolelog.txt)

[StatusIm-2017-04-24-113813.crash.txt](https://github.com/status-im/status-react/files/951279/StatusIm-2017-04-24-113813.crash.txt)
